### PR TITLE
chore: remove jobs

### PR DESCRIPTION
**Pull request by jrandolf-google**
This PR removes the jobs task runner and migrates the doc generator to an internal package.

### CODE REVIEW

File: Herebyfile.mjs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  run: async () => {
+    const content = await readFile('docs/chromium-support.md', {
+      encoding: 'utf8',
+    });
+    const {versionsPerRelease} = await import('./versions.js');
+    const versionsArchived = JSON.parse(
+      await readFile('website/versionsArchived.json', 'utf8')
+    );
+    const buffer = [];
+    for (const [chromiumVersion, puppeteerVersion] of versionsPerRelease) {
```
Comment (OrKoN) [inline Herebyfile.mjs:42]: let's include https://github.com/puppeteer/puppeteer/pull/11108 into this change or as a follow-up? the contributor's CL would go stale with this one.

I have also asked @Lightning00Blade to review as he said he spotted something.

File: docs/browsers-api/index.md [hunk 2]
```diff
@@ -2,34 +2,7 @@
 sidebar_label: API
 ---
 
-# @puppeteer/browsers
```
Comment (Lightning00Blade) [inline docs/browsers-api/index.md]: This should not change, right?

File: packages/puppeteer-core/Herebyfile.mjs [hunk 3]
```diff
@@ -38,8 +38,7 @@ export const generateInjectedTask = task({
       entryPoints: ['src/injected/injected.ts'],
       bundle: true,
       format: 'cjs',
-      platform: 'browser',
-      target: 'ES2022',
+      target: 'chrome117',
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/Herebyfile.mjs]: Let's stick with the previous part as we support Firefox,
I can see someone bumping chrome to new version and that version supporting ESNEXT that Firefox does not and breaking.

File: packages/puppeteer-core/Herebyfile.mjs [hunk 4]
```diff
@@ -52,17 +51,54 @@ export const generateInjectedTask = task({
...
-    await execa('rollup', ['-c', 'rollup.third_party.config.mjs']);
-    await writeFile('lib/esm/package.json', JSON.stringify({type: 'module'}));
+  },
+});
+
+export const buildTask = task({
+  name: 'build',
+  dependencies: [buildTscTask],
+  run: async () => {
+    const packages = ['mitt', 'rxjs'];
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/Herebyfile.mjs]: Can we get the folders and deduce it from there, that ways if we add new one (example `chromium-bidi`) we don't need to add it here?

File: package.json [hunk 5]
```diff
@@ -115,14 +116,10 @@
   },
   "devDependencies": {
     "@actions/core": "1.10.1",
-    "@microsoft/api-documenter": "7.23.9",
     "@microsoft/api-extractor": "7.38.0",
```
Comment (Lightning00Blade) [inline package.json:119]: Why does this still need to live here?

Comment (jrandolf) [reply package.json:119]: API extractor is a separate CLI tool

File: packages/puppeteer-core/Herebyfile.mjs [hunk 6]
```diff
@@ -38,7 +39,7 @@ export const generateInjectedTask = task({
       entryPoints: ['src/injected/injected.ts'],
       bundle: true,
       format: 'cjs',
-      target: 'chrome117',
+      target: ['chrome117', 'firefox113'],
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/Herebyfile.mjs]: Firefox latest stable is 118.

File: packages/puppeteer-core/Herebyfile.mjs [hunk 7]
```diff
@@ -80,23 +81,26 @@ export const buildTask = task({
...
-            outfile: filePath,
-            bundle: true,
-            allowOverwrite: true,
-            format,
-            target: 'node16',
-            minify: true,
-          })
-        );
+    for (const format of formats) {
+      const folder = join('lib', format, 'third_party');
```
Comment (Lightning00Blade) [inline packages/puppeteer-core/Herebyfile.mjs:94]: Nit: We can do this from `src` and filter them. I would make it easier to read.

File: website/docusaurus.config.js [hunk 8]
```diff
@@ -21,12 +21,26 @@ const assert = require('assert');
 
 const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 const lightCodeTheme = require('prism-react-renderer/themes/github');
+const semver = require('semver');
```
Comment (Lightning00Blade) [inline website/docusaurus.config.js:24]: SemVer does not exist on the `website` package, please include it in its `package.json`

**Review by Lightning00Blade — APPROVED** [review]

**Pull request merged**